straxen/plugins/peak_processing.py

```diff
@@ -187,15 +208,25 @@ def compute(self, peaks):
         if not np.sum(peak_mask):
             # Nothing to do, and .predict crashes on empty arrays
             return result
-
-        # Input: normalized hitpatterns in good top PMTs
-        _in = peaks['area_per_channel'][peak_mask, :]
-        _in = _in[:, :self.config['n_top_pmts']][:, self.pmt_mask]
-        with np.errstate(divide='ignore', invalid='ignore'):
-            _in /= _in.sum(axis=1).reshape(-1, 1)
-
-        # Output: positions in mm (unfortunately), so convert to cm
-        _out = self.nn.predict(_in) / 10
+        
+        # XENONnT
+        if self.config['nn_file']:
+            _in = peaks['area_per_channel'][peak_mask, 0:self.config['n_top_pmts']]
+            with np.errstate(divide='ignore', invalid='ignore'):
+                _in = _in / np.max(_in, axis=1).reshape(-1, 1)
```

Comment:
Do you know why this is `np.sum` for 1T?

Reply:
Dividing by sum is a very common way to normalize input. I don't know exactly why it is used in 1T, but using this kinds of expected as it is in almost every machine learning tutorial.
We found that NNs works better on simulation when the input is divided by the maximum value, so models for nT are using `np.max` instead.
It is also possible that normalizing would change if we could find a method that benefits reconstruction more.

Reply:
It's because we were only using relative pattern for training since Xe100 times ;) People like yuehuan and me are not following the advancement in data science. So no real physics reasons for np.sum. I remember that Shixiao showed that nu.max is better, so we are convinced, although I don't understand ;)